packages/selleo-tailwind/tailwind.config.cjs

```diff
@@ -32,6 +32,7 @@ module.exports = {
       7: "96px",
       8: "144px",
       9: "192px",
+      48: "48px",
```

Comment:
weird size, can we try to stick to the extisting spacing or try to adjust design for it?